### PR TITLE
Add missing require

### PR DESCRIPTION
**Pull request by aoby**
https://github.com/code-dot-org/code-dot-org/pull/14780 worked in development, but on staging it failed due to a missing require (dependencies are different in dev and other envs).

CC @caleybrock

### CODE REVIEW

File: lib/forms/pegasus_form_validation.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require_relative 'pegasus_form_errors'
+require 'validates_email_format_of'
```
Comment (ashercodeorg) [inline lib/forms/pegasus_form_validation.rb:2]: That isn't the filename, is it?

Comment (ashercodeorg) [reply lib/forms/pegasus_form_validation.rb:2]: Ah, never mind, it is a gem.

Comment (aoby) [reply lib/forms/pegasus_form_validation.rb:2]: [Sure is](https://github.com/validates-email-format-of/validates_email_format_of/blob/master/lib/validates_email_format_of.rb)

Comment (aoby) [reply lib/forms/pegasus_form_validation.rb:2]: Also if I get the filename wrong, then `dashboard-server` fails to start:

With `require 'i_dont_exist'`:

```
~ bin/dashboard-server

10:05:06 [rerun] Dashboard launched
10:05:09 [rerun] Watching /Users/andrew/Code/code-dot-org/lib, /Users/andrew/Code/code-dot-org/shared/middleware, /Users/andrew/Code/code-dot-org/pegasus for **/*.{rb,ru,yml} (ignoring **/migrations/*.rb) using Darwin adapter
/Users/andrew/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:293:in `require': cannot load such file -- i_dont_exist (LoadError)
	from /Users/andrew/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:293:in `block in require'
	from /Users/andrew/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:259:in `load_dependency'
	from /Users/andrew/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:293:in `require'
	from /Users/andrew/Code/code-dot-org/lib/forms/pegasus_form_validation.rb:2:in `<top (required)>'
...
```

**Pull request merged**